e2e/support/test_support.go

```diff
@@ -1110,6 +1110,79 @@ func CreateTimerKamelet(ns string, name string) func() error {
 	}
 }
 
+func CreateLogKamelet(ns string, name string) func() error {
+	return func() error {
+		kamelet := v1alpha1.Kamelet{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: ns,
+				Name:      name,
+			},
+			Spec: v1alpha1.KameletSpec{
+				Definition: &v1alpha1.JSONSchemaProps{
+					Properties: map[string]v1alpha1.JSONSchemaProp{
+						"loggerName": {
+							Type: "string",
+						},
+					},
+				},
+				Flow: asFlow(map[string]interface{}{
+					"from": map[string]interface{}{
+						"uri": "kamelet:source",
+						"steps": []map[string]interface{}{
+							{
+								"to": "log:{{loggerName}}",
+							},
+						},
+					},
+				}),
+			},
+		}
```

Comment:
Excellent support method. Better to make it a general function such as `CreateKamelet()` and then you use it from another CreateLogKamelet() which is local to your test and is in charge to make a specific behavior.